### PR TITLE
Fix image for syndesis operator olm integration

### PR DESCRIPTION
**Pull request by lgarciaaco**
... and exclude my development files ...

### CODE REVIEW

File: install/operator/deploy/manifests/1.7.0/syndesisoperator.1.7.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -240,8 +240,7 @@ spec:
               serviceAccountName: syndesis-operator
               containers:
                 - name: syndesis-operator
-                  image: lgarciaac/syndesis-operator:latest
-                  #image: syndesis/syndesis-operator:1.7.1-20190607
+                  image: syndesis/syndesis-operator:latest
```
Comment (zregvart) [inline install/operator/deploy/manifests/1.7.0/syndesisoperator.1.7.0.clusterserviceversion.yaml]: Bit odd that we use `latest` on a versioned manifest

Comment (lgarciaaco) [reply install/operator/deploy/manifests/1.7.0/syndesisoperator.1.7.0.clusterserviceversion.yaml]: The final version for 1.7 is not yet build

Comment (zregvart) [reply install/operator/deploy/manifests/1.7.0/syndesisoperator.1.7.0.clusterserviceversion.yaml]: Perhaps we should tag the latest `1.7.*` with `1.7`, I'm a bit afraid that we'll forget to change this from `latest`.

**Comment by lgarciaaco** [comment]
>  @lgarciaaco please make sure that we remember to change from `latest` to a released version when we do release the version; would be good to have this as a check or as a task that `syndesis release` does.

No worries, I wont :)

**Pull request merged**